packages.yml

```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/snapchat_ads_source
-  version: [">=0.7.0", "<0.8.0"]
\ No newline at end of file
+# - package: fivetran/snapchat_ads_source
+#   version: [">=0.8.0", "<0.9.0"]
+
+- git: https://github.com/fivetran/dbt_snapchat_ads_source.git
+  revision: feature/geo_reports
+  warn-unpinned: false
```

Comment:
obligatory reminder to update before merging

Reply:
ditto!

Reply:
same!